CPP/Disk Scheduling Algorithm/FCFS Disk Scheduling.cpp

```diff
@@ -0,0 +1,43 @@
+	
+#include<iostream>
+ 
+using namespace std;
```

Comment:
Do not use `using namespace std`.
[Coding Standards, C++ FAQ](https://isocpp.org/wiki/faq/coding-standards#using-namespace-std)

Instead simply type `std::`.